### ISSUE TITLE
WCAG 4.1.2: Ensures buttons have discernible text (#expandOptionscb67846b-6dd5-448b-ab13-e6351d547800)

### ISSUE THREAD

**Issue by alexr00** [microsoft/vscode-pull-request-github#4191]
Tags: Accessibility, WCAG 4.1.2, button-name

Issue: Ensures buttons have discernible text (button-name - https://accessibilityinsights.io/info-examples/web/button-name)

Element path: .webview;#active-frame;#expandOptionscb67846b-6dd5-448b-ab13-e6351d547800

Snippet: `<button id="expandOptionscb67846b-6dd5-448b-ab13-e6351d547800" class="select-right ">`

How to fix: 
Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

![image](https://user-images.githubusercontent.com/38270282/201960347-6621ec22-9adb-4a4d-9ab8-dadae26859d3.png)
